content/en/docs/Installation/installation-guide/install-with-helm.md

```diff
@@ -80,7 +81,10 @@ create the `kiali-operator` namespace (if needed), and deploy the Kiali
 operator on it.  The `--set cr.create=true` and `--set
 cr.namespace=istio-system` flags instructs to create a Kiali CR in the
 `istio-system` namespace. Since the Kiali CR is created in advance, as soon as
-the Kiali operator starts, it will process it to deploy Kiali.
+the Kiali operator starts, it will process it to deploy Kiali. After Kiali has started,
+you can access Kiali UI through 'http://localhost:20001' by executing 
+`kubectl port-forward service/kiali -n istio-system 20001:20001`
+because of `--set cr.auth.strategy="anonymous"`. But realize that anonymous mode will allow anyone to be able to see and use Kiali. If you wish to require users to authenticate themselves by logging into Kiali, use one of the other [auth strategies](/docs/configuration/authentication/).
```

Comment:
and here, too - should be "cr.spec.auth.strategy"

Reply:
@jmazzitelli sorry to bother you.
you right, i missed it. I'm really sorry.  (https://github.com/kiali/helm-charts/blob/fb5c5f760c9b8d10890620fa28f63924028bd228/kiali-operator/templates/kiali-cr.yaml#L19-L21)

Reply:
I already corrected it. It's live on the site: https://kiali.io/docs/installation/installation-guide/install-with-helm/#install-with-operator

Reply:
I see.
Thanks for your help. 🙇‍♂️